kitsune/settings.py

```diff
@@ -928,7 +924,11 @@ def JINJA_CONFIG():
 
 
 # Google Analytics settings.
-GA_KEY = config('GA_KEY', default='longkey')  # Google API client key
+# GA_KEY is expected b64 encoded.
```

Comment:
GA_KEY is binary. In SCL3 we have in `settings_local.py` which worked OK but we can't put it as is in an env variable. Base64 encoding it does the trick

Reply:
just a heads up, K8s secrets are base64-encoded, so this value will be double base64 encoded. I don't believe there is any impact here, just wanted to note.

```
echo "foo" | base64 | base64 | base64 -D | base64 -D
```